controllers/provider-alicloud/pkg/controller/infrastructure/types.go

```diff
@@ -20,6 +20,12 @@ import (
 )
 
 const (
+	// SeedCloudProviderSecretName is the name of the Secret of Seed's cloud provider credentials
+	SeedCloudProviderSecretName = "vm-image-owner"
```

Comment:
Can we please remove the hard-coded name/namespace here? Also, who creates this secret? I don't see it in the Alicloud extension Helm chart. 
I suggested to provide the credentials to the Alicloud Helm chart, generate a secret object out of it, and make the name/namespace known to alicloud controller (probably via its component config).

Reply:
We added a secret ref in the extension configuration. However,  there is no automatic way to generate such a secret. Even in ls-setup, we can't create a secret in seed cluster because during ls-setup process, seed clusters may not be ready. For current, I would like to as operators to manually create one. New version of OS is critical for AliCloud to release. So shall we just take this solution for this Takt? In the next Takt, I will firgure out a better way.